ffi/mxcfb_sony_h.lua

```diff
@@ -0,0 +1,46 @@
+local ffi = require("ffi")
+
+ffi.cdef[[
+struct mxcfb_rect {
+  unsigned int top;
+  unsigned int left;
+  unsigned int width;
+  unsigned int height;
+};
+
+struct mxcfb_alt_buffer_data {
+  void* virt_addr;
+  unsigned int phys_addr;
+  unsigned int width;
+  unsigned int height;
+  struct mxcfb_rect alt_update_region;
+};
+
+struct mxcfb_update_data {
+  struct mxcfb_rect update_region;
+  unsigned int waveform_mode;
+  unsigned int update_mode;
+  unsigned int update_marker;
+  int temp;
+  unsigned int flags;
+  struct mxcfb_alt_buffer_data alt_buffer_data;
+};
+
+static const int UPDATE_MODE_PARTIAL = 0;
+static const int UPDATE_MODE_FULL = 1;
+
+static const int WAVEFORM_MODE_INIT = 0;
+static const int WAVEFORM_MODE_DU = 1;
+static const int WAVEFORM_MODE_GC16 = 2;
+static const int WAVEFORM_MODE_GC4 = 3;
+static const int WAVEFORM_MODE_A2 = 4;
+static const int WAVEFORM_MODE_AUTO = 257;
+static const int TEMP_USE_AMBIENT = 4096;
+static const int TEMP_USE_AUTO = 4097; // this does not exist, simply use a value different than above
```

Comment:
I'm not quite sure I follow?

Reply:
You mean the comment? I meant there's no actual AUTO constant for temperature in the headers, but looking at the code, if it doesn't receive a AMBIENT setting it will use the temperature sensor. Thus, I simply used a different value for that constant.

Reply:
Okay, thanks! :-)

Reply:
Huh, that's interesting ;).

I'd add the matching 0x1001 define in the header like we do on Kobo/Kindle, just so that it doesn't get zapped if someone decides to re-run gcc-lua one day ;).